pkg/build/webhook/github/github.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 var (
-	ErrNoGitHubEvent = errors.New("missing X-GitHub-Event or X-Gogs-Event")
+	ErrNoGitHubEvent = errors.New("missing X-GitHub-Event or X-Gogs-Event or X-Gitlab-Event")
```

Comment:
please change this text to:
"missing X-GitHub-Event, X-Gogs-Event, or X-Gitlab-Event"